### PR TITLE
DEVPROD-9824 Repeat patch command should work with local modules

### PR DESCRIPTION
**Pull request by bynn**
DEVPROD-9824

### Description
we save local modules in the db so that a repeat patch call will correctly use the same changes 

### Testing
tested on staging 
made a local change on a module and created a [patch](https://spruce-staging.corp.mongodb.com/version/6723de5cd2421f0007fb96e3/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) 
ran `evgs patch-file --diff-patchId 6723de5cd2421f0007fb96e3 --repeat-patch 6723de5cd2421f0007fb96e3 -f`
which created a successful [patch](https://evergreen-staging.corp.mongodb.com/version/6723e569f3450a000747d8a9?redirect_spruce_users=true) with the correct task selected

### CODE REVIEW

File: operations/patch.go [hunk 1]
```diff
@@ -99,6 +99,7 @@ func Patch() cli.Command {
 			mutuallyExclusiveArgs(false, patchDescriptionFlagName, autoDescriptionFlag),
 			mutuallyExclusiveArgs(false, preserveCommitsFlag, uncommittedChangesFlag),
 			mutuallyExclusiveArgs(false, repeatDefinitionFlag, repeatPatchIdFlag),
+			mutuallyExclusiveArgs(false, repeatPatchIdFlag, includeModulesFlag),
```
Comment (hadjri) [inline operations/patch.go:102]: For my understanding why do these have to be mutually exclusive?

Comment (hadjri) [inline operations/patch.go:102]: For my understanding why do these have to be mutually exclusive?

Comment (bynn) [reply operations/patch.go:102]: when we use repeat patch, we use the changes from that patch including the modules so it wouldn't make sense to include another modules!

**Review by hadjri — APPROVED** [review]
nit: uptick cli

**Pull request merged**